aep/general/0004/aep.md.j2

```diff
@@ -64,19 +64,25 @@ message Topic {
 
 {% tab oas %}
 
-For OpenAPI 3.0, use the `x-aep-resource` extension:
+For OpenAPI 3.0, Resources must be defined in `#components/schemas` and use the `x-aep-resource` extension:
```

Comment:
Some people may be hard-coding the schema into their path objects - is increasing friction for those customers a concern ?

OTOH I could imagine it's not too hard to actually transform the schema reference.

Reply:
Especially if the OAS is being hand-written, I think hard-coding the full schema is an anti pattern. The multiple references will eventually get out of date. 

Using a schema is the only way I can think of to ensure users have updated all of the affected APIs.